### ISSUE TITLE
Test datasets

### ISSUE THREAD

**Issue by aminpdik** [niejiahao1998/IFA#14]
Thank you for the excellent paper!

I have a few questions regarding the test datasets to ensure I’m following the correct procedure:

DeepGlobe: For the DeepGlobe dataset, I am using the preprocessed DeepGlobe link instead of the main dataset link. In the preprocessed version, there are six folders per class, but the total number of images across these folders exceeds the 5,666 images reported in the original ECCV paper for CDFSS. Could you confirm if I’m using the correct images for testing on DeepGlobe?

FSS-1000: I reviewed your code multiple times and noticed you are using split='val' for the dataloader when testing. However, the original ECCV CDFSS paper states that the test split of FSS-1000 was used to report the results. Could you clarify this discrepancy?

**Comment by niejiahao1998** [comment]
Hi,
Thanks for your interest!

1-The details of the processed Deepglobe dataset:
I followed the processing code to process the data instead of directly using the data provided by the ECCV repo.
Consequently, I am not sure if these data are strictly the same.
I provide you the number of images in each category, and you may have a check.
1-6891, 2-4189, 3-4333, 4-1564, 5-1370, and 6-2195.

2-The dataloader of FSS-1000:
I strictly followed the same setting with the ECCV repo.
If there are slight differences between the paper and the code, I recommend you check the ECCV code.

**Comment by aminpdik** [comment]
Thanks @niejiahao1998 

I generated Deepglobe using the provided code, and the number of images is the same as what you mentioned.

I also checked the ECCV code for FSS-1000. This is the dataloader for all test data.

FSSDataset.initialize(img_size=400, datapath=args.datapath)
dataloader_test = FSSDataset.build_dataloader(args.benchmark, args.bsz, args.nworker, args.fold, 'test', args.nshot)

As you can see, they are using 'test' split for FS1000 and other test datasets. But in your code, it seems you are using 'val' split, which means the fss will be tested on the validation set, not the test set.
